### PR TITLE
amplifyconfiguration.json -> amplify_outputs.json

### PR DESCRIPTION
**Pull request by josefaidt**
#### Description of changes:

follow-up to https://github.com/aws-amplify/docs/pull/7277

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/gen2/build-a-backend/add-aws-services/rest-api/index.mdx [hunk 1]
```diff
@@ -73,7 +73,7 @@ To initialize the REST API you need to configure Amplify with `Amplify.configure
 
 ```tsx title="pages/index.tsx"
 import { Amplify } from 'aws-amplify';
-import amplifyconfig from './src/amplifyconfiguration.json';
+import amplifyconfig from '../amplify_outputs.json';
 Amplify.configure(amplifyconfig):
```
Comment (Amplifiyer) [inline src/pages/gen2/build-a-backend/add-aws-services/rest-api/index.mdx:77]:
```suggestion
import amplifyoutputs from '../amplify_outputs.json';
Amplify.configure(amplifyoutputs):
```

File: src/pages/gen2/build-a-backend/server-side-rendering/index.mdx [hunk 2]
```diff
@@ -29,13 +29,13 @@ npm add aws-amplify @aws-amplify/adapter-nextjs
 
 You will need to create a `runWithAmplifyServerContextRunner` function to use Amplify APIs on the server-side of your Next.js app.
 
-You can create an `amplifyServerUtils.ts` file under a `utils` folder in your codebase. In this file, you will import the Amplify configuration object from the `amplifyconfiguration.json` file that is generated by the Amplify CLI, and use the `createServerRunner` function to create the `runWithAmplifyServerContextRunner` function.
+You can create an `amplifyServerUtils.ts` file under a `utils` folder in your codebase. In this file, you will import the Amplify configuration object from the `amplify_outputs.json` file that is generated by the Amplify CLI, and use the `createServerRunner` function to create the `runWithAmplifyServerContextRunner` function.
```
Comment (Amplifiyer) [inline src/pages/gen2/build-a-backend/server-side-rendering/index.mdx:32]:
```suggestion
You can create an `amplifyServerUtils.ts` file under a `utils` folder in your codebase. In this file, you will import the Amplify backend outputs from the `amplify_outputs.json` file that is generated by the Amplify CLI, and use the `createServerRunner` function to create the `runWithAmplifyServerContextRunner` function.
```

Is `Amplify CLI` the right wording here?

File: src/pages/gen2/build-a-backend/add-aws-services/rest-api/index.mdx (hunk 1 above)
Comment (josefaidt) [reply src/pages/gen2/build-a-backend/add-aws-services/rest-api/index.mdx]: good call, updated all references to this import

File: src/pages/gen2/build-a-backend/server-side-rendering/index.mdx (hunk 2 above)
Comment (josefaidt) [reply src/pages/gen2/build-a-backend/server-side-rendering/index.mdx]: > Is Amplify CLI the right wording here?

it is not

File: src/pages/gen2/build-a-backend/add-aws-services/rest-api/index.mdx [hunk 3]
```diff
@@ -73,7 +73,7 @@ To initialize the REST API you need to configure Amplify with `Amplify.configure
 
 ```tsx title="pages/index.tsx"
 import { Amplify } from 'aws-amplify';
-import amplifyconfig from './src/amplifyconfiguration.json';
+import outputs from '../amplify_outputs.json';
 Amplify.configure(amplifyconfig):
```
Comment (reesscot) [inline src/pages/gen2/build-a-backend/add-aws-services/rest-api/index.mdx:77]:
```suggestion
Amplify.configure(outputs):
```

File: src/pages/gen2/build-a-backend/data/connect-to-API/index.mdx [hunk 4]
```diff
@@ -32,11 +32,11 @@ npm add aws-amplify
...
 
-When you deploy you're iterating on your backend (`npx amplify sandbox`), an `amplifyconfiguration.json` file is generated for you. This file contains your API's endpoint information and auth configurations. Add the following code to your app's entrypoint to initialize and configure the Amplify client library:
+When you deploy you're iterating on your backend (`npx amplify sandbox`), an `amplify_outputs.json` file is generated for you. This file contains your API's endpoint information and auth configurations. Add the following code to your app's entrypoint to initialize and configure the Amplify client library:
 
 ```ts
 import { Amplify } from 'aws-amplify';
-import config from '../amplifyconfiguration.json';
+import outputs from '../amplify_outputs.json';
 
 Amplify.configure(config);
```
Comment (josefaidt) [inline src/pages/gen2/build-a-backend/data/connect-to-API/index.mdx:41]:
```suggestion
Amplify.configure(outputs);
```

File: src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx [hunk 5]
```diff
@@ -438,7 +438,7 @@ The `Authenticator` component offers a simple way to add authentication flows in
 
   import { Amplify } from 'aws-amplify';
   import config from './amplifyconfiguration.json';
-  Amplify.configure(config);
+  Amplify.configure(outputs);
```
Comment (reesscot) [inline src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:441]: This is in gen1 docs and shouldn't change

File: src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx [hunk 6]
```diff
@@ -376,7 +376,7 @@ import type { WithAuthenticatorProps } from '@aws-amplify/ui-react';
 import { withAuthenticator } from '@aws-amplify/ui-react';
 import '@aws-amplify/ui-react/styles.css';
 import config from './amplifyconfiguration.json';
-Amplify.configure(config);
+Amplify.configure(outputs);
```
Comment (reesscot) [inline src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:379]: This is in gen1 docs and shouldn't change

File: src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx [hunk 7]
```diff
@@ -158,7 +158,7 @@ In your app's entry point (specifically, **App.js**, **index.js**, **\_app.js**,
 ```javascript
 import { Amplify } from 'aws-amplify';
 import config from './amplifyconfiguration.json';
-Amplify.configure(config);
+Amplify.configure(outputs);
```
Comment (reesscot) [inline src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:161]: This is in gen1 docs and shouldn't change

File: src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx [hunk 8]
```diff
@@ -399,7 +399,7 @@ import { Amplify } from 'aws-amplify';
 import { withAuthenticator } from '@aws-amplify/ui-react';
 import '@aws-amplify/ui-react/styles.css';
 import config from './amplifyconfiguration.json';
-Amplify.configure(config);
+Amplify.configure(outputs);
```
[the remark this answers is not in the document]
Comment (josefaidt) [reply src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:402]: oh gosh darnit, I was using the same regex from when I was pointing to next-release... let me update this entire PR. good catch!

[the remark this answers is not in the document]
Comment (josefaidt) [reply src/pages/[platform]/build-a-backend/auth/set-up-auth/index.mdx:402]: ah good catch, borked my regex

**Review by renebrandel — APPROVED** [review]

File: src/pages/gen2/start/quickstart/vite-react-app/index.mdx [hunk 9]
```diff
@@ -109,7 +109,7 @@ Open `src/main.tsx` and add the following code below the last import:
 
 ```tsx title="src/main.tsx"
 import { Amplify } from 'aws-amplify';
-import amplifyconfig from '../amplifyconfiguration.json';
+import outputs from '../amplify_outputs.json';
 
 Amplify.configure(amplifyconfig);
```
Comment (renebrandel) [inline src/pages/gen2/start/quickstart/vite-react-app/index.mdx:114]:
```suggestion
Amplify.configure(outputs);
```

**Review by renebrandel — APPROVED** [review]

File: src/pages/gen2/build-a-backend/data/connect-from-server-runtime/index.mdx [hunk 10]
```diff
@@ -96,7 +96,7 @@ To generate a Data client for the Next.js server runtime using `NextRequest` and
 import { type Schema } from '@/amplify/data/resource';
 import { createServerRunner } from '@aws-amplify/adapter-nextjs';
 import { generateServerClientUsingReqRes } from '@aws-amplify/adapter-nextjs/data';
-import amplifyConfig from '@/amplifyconfiguration.json';
+import outputs from '@/amplify_outputs.json';
```
Comment (reesscot) [inline src/pages/gen2/build-a-backend/data/connect-from-server-runtime/index.mdx:99]: where's the corresponding use?

File: src/pages/gen2/build-a-backend/data/connect-from-server-runtime/index.mdx [hunk 11]
```diff
@@ -130,7 +130,7 @@ Import the cookie-based server Data client in your Next.js React Server Componen
 ```ts
 import { type Schema } from '@/amplify/data/resource';
 import { generateServerClientUsingCookies } from '@aws-amplify/adapter-nextjs/data';
-import amplifyConfig from '@/amplifyconfiguration.json';
+import outputs from '@/amplify_outputs.json';
 import { cookies } from 'next/headers';
```
Comment (reesscot) [inline src/pages/gen2/build-a-backend/data/connect-from-server-runtime/index.mdx:134]: missed updating amplifyconfig below

File: src/pages/gen2/build-a-backend/data/customize-authz/public-data-access/index.mdx [hunk 12]
```diff
@@ -64,12 +64,12 @@ const schema = a.schema({
...
 
 <Callout info>
-If you're not using the auto-generated **amplifyconfiguration.json** file, then you must set the Amplify Library resource configuration's `allowGuestAccess` flag to `true`. This lets the Amplify Library use the unauthenticated role from your Cognito identity pool when your user isn't logged in.
+If you're not using the auto-generated **amplify_outputs.json** file, then you must set the Amplify Library resource configuration's `allowGuestAccess` flag to `true`. This lets the Amplify Library use the unauthenticated role from your Cognito identity pool when your user isn't logged in.
 
 <Accordion title="Amplify configuration">
 ```ts title="src/App.tsx"
 import { Amplify } from "aws-amplify";
-import config from "../amplifyconfiguration.json";
+import outputs from "../amplify_outputs.json";
```
Comment (reesscot) [inline src/pages/gen2/build-a-backend/data/customize-authz/public-data-access/index.mdx:72]: misisng use of outputs below (I see a ...config)

File: src/pages/gen2/build-a-backend/server-side-rendering/index.mdx [hunk 13]
```diff
@@ -29,13 +29,13 @@ npm add aws-amplify @aws-amplify/adapter-nextjs
...
 
 For example, the `utils/amplifyServerUtils.ts` file may contain the following content:
 
 ```typescript
 import { createServerRunner } from '@aws-amplify/adapter-nextjs';
-import config from '@/amplifyconfiguration.json';
+import outputs from '@/amplify_outputs.json';
 
 export const { runWithAmplifyServerContext } = createServerRunner({
   config
```
Comment (reesscot) [inline src/pages/gen2/build-a-backend/server-side-rendering/index.mdx:41]:
```suggestion
  outputs
```

File: src/pages/gen2/build-a-backend/server-side-rendering/index.mdx [hunk 14]
```diff
@@ -61,7 +61,7 @@ When you use the Amplify library on the client-side of your Next.js app, you wil
 ```typescript
 'use client';
 
-import config from '@/amplifyconfiguration.json';
+import outputs from '@/amplify_outputs.json';
 import { Amplify } from 'aws-amplify';
 
 Amplify.configure(config, {
```
Comment (reesscot) [inline src/pages/gen2/build-a-backend/server-side-rendering/index.mdx:67]:
```suggestion
Amplify.configure(outputs, {
```

File: src/pages/gen2/build-a-backend/server-side-rendering/index.mdx [hunk 15]
```diff
@@ -93,7 +93,7 @@ If you're using the Next.js App Router, you can create a client component to con
 'use client';
 
 import { Amplify } from 'aws-amplify';
-import config from '../amplifyconfiguration.json';
+import outputs from '../amplify_outputs.json';
 
 Amplify.configure(config, { ssr: true });
```
Comment (reesscot) [inline src/pages/gen2/build-a-backend/server-side-rendering/index.mdx:98]:
```suggestion
Amplify.configure(outputs, { ssr: true });
```

File: src/pages/gen2/start/quickstart/nextjs-app-router-client-components/index.mdx [hunk 16]
```diff
@@ -48,7 +48,7 @@ Next, create a `components` folder in the root of your project and the contents
 // components/ConfigureAmplify.tsx
 "use client";
 
-import config from "@/amplifyconfiguration.json";
+import outputs from "@/amplify_outputs.json";
 import { Amplify } from "aws-amplify";
 
 Amplify.configure(config, { ssr: true });
```
Comment (reesscot) [inline src/pages/gen2/start/quickstart/nextjs-app-router-client-components/index.mdx:54]:
```suggestion
Amplify.configure(outputs, { ssr: true });
```

File: src/pages/gen2/start/quickstart/nextjs-app-router-server-components/index.mdx [hunk 17]
```diff
@@ -51,7 +51,7 @@ Next, create a `components` folder in the root of your project and copy the cont
 
 import { Amplify } from "aws-amplify";
 
-import config from "@/amplifyconfiguration.json";
+import outputs from "@/amplify_outputs.json";
 
 Amplify.configure(config, { ssr: true });
```
Comment (reesscot) [inline src/pages/gen2/start/quickstart/nextjs-app-router-server-components/index.mdx:56]:
```suggestion
Amplify.configure(outputs, { ssr: true });
```

File: src/pages/gen2/start/quickstart/nextjs-app-router-server-components/index.mdx [hunk 18]
```diff
@@ -113,7 +113,7 @@ import { generateServerClientUsingCookies } from "@aws-amplify/adapter-nextjs/ap
 import { getCurrentUser } from "aws-amplify/auth/server";
 
 import { type Schema } from "@/amplify/data/resource";
-import config from "@/amplifyconfiguration.json";
+import outputs from "@/amplify_outputs.json";
 
 export const { runWithAmplifyServerContext } = createServerRunner({
   config,
```
Comment (reesscot) [inline src/pages/gen2/start/quickstart/nextjs-app-router-server-components/index.mdx:119]:
```suggestion
  outputs,
```

File: src/pages/gen2/build-a-backend/data/customize-authz/public-data-access/index.mdx (hunk 12 above)
Comment (reesscot) [reply src/pages/gen2/build-a-backend/data/customize-authz/public-data-access/index.mdx:72]: pushed up a fix

**Review by reesscot — APPROVED** [review]

**Review by renebrandel — APPROVED** [review]

**Pull request merged**